tasks/Module 4/task_1.py

```diff
@@ -0,0 +1,41 @@
+"""Task 1 of Module 4."""
+from pathlib import Path
+import pyinputplus as pyip
+
+WORDS_TO_REPLACE = ["ADJECTIVE", "NOUN", "VERB"]
+
+
+def mad_libs():
+    """
+    Create and read in text files
+    and lets the user add their own text instead of words in constant.
+    """
+    input_file_path = Path("file.txt")
+    output_file_path = Path("output_file.txt")
+    new_text = []
+
+    if not input_file_path.is_file():
+        with open(input_file_path, "w") as input_file:
+            input_file.write("The ADJECTIVE panda walked to the NOUN and then VERB. "
+                             "A nearby NOUN was unaffected by these events.")
+
+    with open(input_file_path, "r") as input_file:
```

Comment:
You can combine this into one with statement:
```python
with open(input_file_path, "r") as input_file, open(output_file_path, "w") as output_file:
```

Reply:
The code is changed based on above